src/test/java/org/llorllale/cactoos/matchers/MatchesTest.java

```diff
@@ -58,41 +54,15 @@ void matches() {
      * Gives negative testing result for the invalid arguments.
      */
     @Test
-    void matchStatus() {
+    void mismatches() {
```

Comment:
@victornoel What about the match case.

Reply:
@baudoliver7 I will suppose you mean testing `Matches` using `Matches`? it's a good idea

Reply:
@victornoel Yes